rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -345,7 +342,6 @@ std::string SPUThread::dump() const
 
 void SPUThread::cpu_init()
 {
-	gpr = {};
```

Comment:
Removing GPR cleanup is pretty bad hack, and may cause a lot of trouble in future.

Reply:
registers that are not the arguments of the threaded spus are undefined iirc, it boosts fps a bit on "The Eye Of Judgment" which constanly executes an spu group join, stop 0x101 and spu group startfunctions

Reply:
now it only clears the GPR in threaded spus, raw spus certainly dont need it